android/filament-utils-android/src/main/java/com/google/android/filament/utils/Manipulator.java

```diff
@@ -267,7 +267,7 @@ public Builder flightMoveDamping(float damping) {
          * @return this <code>Builder</code> object for chaining calls
          */
         @NonNull
-        public Builder groundPlane(float a, float b, float c, float d) {
```

Comment:
Was this intentional?

Reply:
Grrr, that's CLion's refactoring tools that do things where they are not supposed to. Good catch.

Reply:
I renamed a variable called `groundPlane` in C++ code using the rename refactor, and it renamed a Java method. Wth.